fairscale/nn/model_parallel/cross_entropy.py

```diff
@@ -64,8 +64,7 @@ def forward(ctx, vocab_parallel_logits, target):  # type: ignore
         )
 
         # Sum of exponential of logits along vocab dimension across all GPUs.
-        exp_logits = vocab_parallel_logits
-        torch.exp(vocab_parallel_logits, out=exp_logits)
+        exp_logits = vocab_parallel_logits.exp()
```

Comment:
```
RuntimeError: Output 0 of _ScatterToModelParallelRegionBackward is a view and its base or another view of its base has been modified inplace. This view was created inside a custom Function (or because an input was returned as-is) and the autograd logic to handle view+inplace would override the custom backward associated with the custom Function, leading to incorrect gradients. This behavior is forbidden. You can remove this warning by cloning the output of the custom Function.
I believe because of https://github.com/pytorch/pytorch/pull/56093
```

Reply:
maybe skip the failing test when torch version >=1.9? Add a comment here for TODO and file a new issue so that we don't forget?